controller/cluster/controller.go

```diff
@@ -65,6 +65,12 @@ func (c *clusterController) onUpdate(oldObj, newObj interface{}) {
 
 	logClusterEventBegin(newCluster, ClusterUpdated)
 
+	err := c.clusterOperator.UpdateCluster(newCluster)
+	if err != nil {
+		logging.LogCluster(newCluster).WithField("event", ClusterUpdated).Error(err)
+		return
```

Comment:
atm hooks should always run so don't return here and wrap logSuccess in an `else` statement

Reply:
Done.